frontend/src/metabase/css/reset.module.css

```diff
@@ -0,0 +1,63 @@
+/** CSS reset scoped to the .mb-wrapper class so we can use it in the SDK */
```

Comment:
maybe we should drop the `.module` parte as this is a "global" css even though it's scoped to .mb-wrapper